FeedAPIChallenge/Feed Feature/FeedImageMapper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by admin on 22/08/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedImageMapper {
```

Comment:
The convention in this codebase is to omit the default access level keyword of `internal`, please remove it.
```suggestion
final class FeedImageMapper {
```